ca_description/urdf/create_light_sensor.xacro

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://ros.org/wiki/xacro">
+
+<xacro:macro name="light_sensor" params="prefix *origin *optical_origin">
+
+    <xacro:include filename="$(find ca_description)/urdf/common_properties.xacro"/>
+    
+    <xacro:property name="name" value="${prefix}_light_sensor"/>
+    <xacro:property name="parent_link" value="base_link"/>
+
+
+        <joint name="${name}_joint" type="fixed">
+            <xacro:insert_block name="origin"/>
+            <parent link="${parent_link}" />
+            <child link="${name}_link" />
+        </joint>
+
+        <link name="${name}_link">
+        <visual>
+            <geometry>
+                <mesh filename="package://ca_description/meshes/last_lens.stl" />
+            </geometry>
+            <material name="blue">
```

Comment:
I don't think this declaration here is necessary. Please double check.

Reply:
You already declared its color somewhere else.

Reply:
If I don't add this line, the color in RViz changes. I mean, the color is not so important in RViz, but I dont quite understand why it happens this way.

Reply:
Oh! That happens because `material` in this case is a ROS parameter and in the other part of the description, it's Gazebo's. So, leave it as it is now.